inspirehep/modules/editor/api.py

```diff
@@ -248,6 +248,7 @@ def get_workflow_and_schema(workflow_id):
         "workflow": {
             "id": workflow.id,
             "_extra_data": workflow.extra_data,
+            "_workflow": workflow.workflow,
```

Comment:
`workflow._workflow`, you're copying ES `hits.hits` :smile:

Reply:
well there was no option tbh, I had to imitate the `/api/holdingpen/<id>`